src/lamp_py/tableau/jobs/glides.py

```diff
@@ -190,15 +190,20 @@ def create_parquet(self, _: None) -> None:
         self.update_parquet(None)
 
     def update_parquet(self, _: None) -> bool:
-        # only run once per day after 11AM UTC - 6 or 7 AM EST
+        if self.first_run:  # type: ignore
+            self.first_run = False
+            create_trips_updated_glides_parquet(self, None)
+            return True
+
+        # only run once per day after 10AM UTC - 5 or 6 AM EST
         if object_exists(tableau_glides_all_trips_updated.s3_uri):
             now_utc = datetime.now(tz=timezone.utc)
             last_mod: datetime = file_list_from_s3_with_details(
                 bucket_name=tableau_glides_all_trips_updated.bucket,
                 file_prefix=tableau_glides_all_trips_updated.prefix,
             )[0]["last_modified"]
```

Comment:
If the goal is to get this to run on every call to `start_parquet_updates` it makes much more sense to delete, or at least comment out, this logic that is attempting to only run this job once per day. 

Having unreachable code will be very confusing for anyone having to update this in the future.